src/utils/display.ts

```diff
@@ -58,9 +68,19 @@ export const makeSignCanonAddrRegistryEntryContent = (
 
 export const makeSignOperationContent = (
   opMetadata: OperationMetadata,
-  erc20s: Map<string, Erc20Config>
+  erc20s: Map<string, Erc20Config>,
+  gasAssetContractAddr: Address,
+  fee: bigint
 ): Panel => {
-  const items = opMetadata.items.map((item) => {
+  const headItem = {
+    heading: "Confirm transaction from our Nocturne account".replace(
```

Comment:
```suggestion
    heading: "Confirm transaction from your Nocturne account".replace(
```
![image](https://github.com/nocturne-xyz/snap/assets/21699995/b4c41d5b-681a-415d-83dd-f07cdfe13d23)
"our account"